pkgs/by-name/tr/treefmt/functions-doc.nix

```diff
@@ -0,0 +1,57 @@
+{
+  lib,
+  writers,
+  nixdoc,
+  runCommand,
+  treefmt,
+}:
+let
+  root = toString ./.;
+  revision = lib.trivial.revisionWithDefault "master";
```

Comment:
@philiptaron have I stumbled on the same footgun that you solved in https://github.com/NixOS/nixpkgs/pull/330915 ?

This build is a new transitive dependency of nixpkgs-manual as of this PR.